### PR TITLE
Circleci multi deploy

### PR DESCRIPTION
**Pull request by asmega**
- On merge into master CircleCI now deploys into test
- With approval, CircleCI can now deploy to integration and live

### CODE REVIEW

**Comment by asmega** [comment]
Travis failures are to do with travis.org vs travis.com. we are setup for .org but not .com where it fails. also we are moving over to circle so have started disabling travis runs

File: scripts/build_and_push_all.sh [hunk 1]
```diff
@@ -1,26 +1,23 @@
...
-  export AWS_SECRET_ACCESS_KEY=${!SECRET_VAR_NAME}
+  AWS_ACCESS_KEY_ID_ENCODED=`kubectl -n formbuilder-repos get secrets ecr-repo-fb-submitter-$TYPE -o jsonpath='{.data.access_key_id}'`
+  AWS_SECRET_ACCESS_KEY_ENCODED=`kubectl -n formbuilder-repos get secrets ecr-repo-fb-submitter-$TYPE -o jsonpath='{.data.secret_access_key}'`
+
+  if [ `uname` == 'Darwin' ]
+  then
+    export AWS_ACCESS_KEY_ID=`echo $AWS_ACCESS_KEY_ID_ENCODED | base64 --decode`
+    export AWS_SECRET_ACCESS_KEY=`echo $AWS_SECRET_ACCESS_KEY_ENCODED | base64 --decode`
+  else
+    export AWS_ACCESS_KEY_ID=`echo $AWS_ACCESS_KEY_ID_ENCODED | base64 -d`
```
Comment (emileswarts) [inline scripts/build_and_push_all.sh]: Any reason we prefer shorthand decode flag when not on Darwin?
If both used --decode then we could get rid of this conditional?

Comment (asmega) [reply scripts/build_and_push_all.sh]: unfortunately the api is different on mac and alpine

mac:
```
Usage:	base64 [-hvD] [-b num] [-i in_file] [-o out_file]
  -h, --help     display this message
  -D, --decode   decodes input
  -b, --break    break encoded string into num character lines
  -i, --input    input file (default: "-" for stdin)
  -o, --output   output file (default: "-" for stdout)
```

alpine (fb-builder):
```
/ # echo "foo" | base64 --decode
base64: unrecognized option: decode
BusyBox v1.29.3 (2019-01-24 07:45:07 UTC) multi-call binary.

Usage: base64 [-d] [FILE]

Base64 encode or decode FILE to standard output
	-d	Decode data
```

Comment (emileswarts) [reply scripts/build_and_push_all.sh]: Oh I see, thanks.

**Pull request merged**